spec/helpers.rb

```diff
@@ -222,7 +222,7 @@ def create_and_review_incident
 
   def submit_to_state
     visit_status :state_submission
-    click_button 'FINAL SUBMISSION'
+    click_button 'SUBMISSION'
```

Comment:
Can we change the specs or whatever so that it always says "FINAL SUBMISSION"? I like the word "final" since it stresses the.... well, finality of clicking the button.